generics/services/sunbird.js

```diff
@@ -0,0 +1,177 @@
+/**
```

Comment:
@VISHNUDAS-tunerlabs currently we have USER_SERVICE_URL & users.js can we add all the things in the existing file instead of creating sunbird.js